docs/source/pickling_cuml_models.ipynb

```diff
@@ -0,0 +1,781 @@
+    "\n",
+    "cluster = LocalCUDACluster()\n",
+    "client = Client(cluster)\n",
+    "client"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will use `make_blobs` from the `cuml.dask.datasets` package to create create a distributed dataset of clustered points for this example. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from cuml.dask.datasets import make_blobs\n",
+    "\n",
+    "n_workers = len(client.scheduler_info()[\"workers\"].keys())\n",
+    "\n",
+    "X, y = make_blobs(n_samples=5000, \n",
+    "                  n_features=30,\n",
+    "                  centers=5, \n",
+    "                  cluster_std=0.4, \n",
+    "                  random_state=0,\n",
+    "                  n_parts=n_workers*5)\n",
+    "\n",
+    "X = X.persist()\n",
+    "y = y.persist()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We verify the distribution of the data across the Dask workers"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'tcp://127.0.0.1:46298': (), 'tcp://127.0.0.1:46704': ()}"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "client.has_what()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Train Distributed Model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from cuml.dask.cluster import KMeans\n",
+    "\n",
+    "dist_model = KMeans(n_clusters=5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The combined model returns `None` when not yet trained"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "None\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(dist_model.get_combined_model())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<cuml.dask.cluster.kmeans.KMeans at 0x7f98253318e0>"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "dist_model.fit(X)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When an attribute doesn't exist on the distributed model, it is proxied to the underlying single-GPU models.\n",
+    "\n",
+    "Note: If needed, we could move the proxy logic into a function that can be overrided in subclasses for special cases (e.g., random forests). "
```

Comment:
Maybe too advanced of a topic to mention in a fairly introductory notebook? Just trying to be very minimal.